UW_template/CCEP_Mayo/sub-01/ses-01/ieeg/sub-01_ses-01_task-CCEP_run-01_events.tsv

```diff
@@ -0,0 +1,2 @@
+onset	duration	trial_type	response_time	stim_file	HED
```

Comment:
Can you format this in the same manner as the Mayo template? As you can see in the example, the _events.tsv is accompanied by an _events.json describing the extra columns. This is BIDS compatible and it is essential that we keep the naming consistent across sites.